server/events/tests/shared_case.py

```diff
@@ -0,0 +1,24 @@
+from datetime import timedelta
```

Comment:
Don't import classes from modules, a lot of development environments don't have smart-lookup, so it's helpful to import the specific module you need, and then use the class when necessary. i.e.

```
import datetime
# ...
something = datetime.timedelta(...)
```

Reply:
fixed